app/models/spree/tax_rate_decorator.rb

```diff
@@ -30,11 +24,6 @@ def adjust(order, item)
     raise SpreeAvatax::TaxRateInvalidOperation.new("Spree::TaxRate#adjust should never be called when Avatax is present")
   end
 
-  def compute_amount(item)
-    # Avatax tax adjustments should always be finalized so Spree should never attempt to call this code
-    raise SpreeAvatax::TaxRateInvalidOperation.new("Spree::TaxRate#compute_amount should never be called when Avatax is present")
-  end
```

Comment:
For the time being, that has to stay as it might be called from adjustments: https://github.com/solidusio/solidus/blob/master/core/app/models/spree/adjustment.rb#L164